src/ucm/cuda/cudamem.c

```diff
@@ -456,13 +456,24 @@ static void ucm_cudamem_get_existing_alloc(ucm_event_handler_t *handler)
 
 ucs_status_t ucm_cuda_get_current_device_info(ucs_sys_bus_id_t *bus_id)
 {
+    static ucs_sys_bus_id_t cached_bus_id = {0xffff, 0xff, 0xff, 0xff};
     CUresult cu_err;
     CUdevice cuda_device;
     CUpointer_attribute attribute;
     int attr_result;
 
     ucm_trace("ucm_cuda_get_current_device_info");
 
+    if ((cached_bus_id.domain == 0xffff) &&
```

Comment:
can we simply this check by just checking one filed (slot or function)

Reply:
You're right. Just domain should suffice.